bigchaindb/commands/bigchaindb.py

```diff
@@ -317,41 +315,38 @@ def create_parser():
                                help='The backend to use. It can only be '
                                '"localmongodb", currently.')
 
-    # parser for managing validator elections
-    validator_parser = subparsers.add_parser('upsert-validator',
-                                             help='Add/update/delete a validator.')
-
-    validator_subparser = validator_parser.add_subparsers(title='Action',
-                                                          dest='action')
-
-    new_election_parser = validator_subparser.add_parser('new',
-                                                         help='Calls a new election.')
+    # parser for managing elections
+    election_parser = subparsers.add_parser('election',
+                                            help='Manage elections.')
 
-    new_election_parser.add_argument('public_key',
-                                     help='Public key of the validator to be added/updated/removed.')
+    election_subparser = election_parser.add_subparsers(title='Action',
+                                                        dest='action')
 
-    new_election_parser.add_argument('power',
-                                     type=int,
-                                     help='The proposed power for the validator. '
-                                          'Setting to 0 will remove the validator.')
+    new_election_parser = election_subparser.add_parser('new',
```

Comment:
We dropped the [idea of using `new` keyword](https://github.com/bigchaindb/BEPs/pull/82/files#diff-ea17bcdbca54e1a9cd9a2705c65d9c64R128).